.github/workflows/regular-build.yml

```diff
@@ -0,0 +1,163 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+name: Build Regularly
```

Comment:
Seems like this action is build for clients, So i would say it is `Build Clients` Or `Clients CI` and not `Build Regularly`.

Reply:
Oh, I just note that this also build  for pegasus server, so it is great, but also `Build all` would be great,  let the name can tell the people what is it doing.

Reply:
Good idea, I've updated it to `Lint and build regularly`, I think `all` is the default behavior it should be so I omit it.